sdk/lib/3rdparty/zlib/contrib/minizip/zip.c

```diff
@@ -519,15 +519,17 @@ local ZPOS64_T zip64local_SearchCentralDir(const zlib_filefunc64_32_def* pzlib_f
       break;
 
     for (i=(int)uReadSize-3; (i--)>0;)
+    {
       if (((*(buf+i))==0x50) && ((*(buf+i+1))==0x4b) &&
         ((*(buf+i+2))==0x05) && ((*(buf+i+3))==0x06))
       {
         uPosFound = uReadPos+i;
         break;
       }
+    }
 
-      if (uPosFound!=0)
-        break;
+    if (uPosFound!=0)
+      break;
```

Comment:
Meaningful code style change?
If not, please revert.

Reply:
GCC 8 complains on such things by default. See commit message
https://github.com/reactos/reactos/pull/1543/commits/a75d92714e99505b3639effb69c6b89bdacc05c6

Reply:
Can't we just drop that warning for 3rd party code?